src/js/menu/operations/TurfOperation.js

```diff
@@ -10,7 +11,16 @@ L.DNC.TurfOperation = L.DNC.Operation.extend({
     },
 
     execute: function () {
-        var layers = L.DNC.layerlist.selection.list;
+        /*
+        **
+        **  TODO: this is the type of referencing
+        **  that feels like it should be avoided.
+        **  it is the only reference left and it cannot
+        **  be factored out until we potentially
+        **  revisit how MenBar, Menu and Operation work together
```

Comment:
While we don't have to solve it all right now, can you give some hints in regards to what you're thinking about a refactor between the MenuBar, Menu, and Operation?